app/helpers/active_storage/meta_helper.rb

```diff
@@ -0,0 +1,7 @@
+module ActiveStorage::MetaHelper
+  def active_storage_meta_tags
+    safe_join [
+      tag.meta(name: 'direct-uploads-url', content: rails_direct_uploads_url)
+    ]
```

Comment:
That's supported currently and will take precedence over the the meta tag value.